addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,55 @@
+# iCloud Binding
+
+The iCloud Binding helps to integrate apple device information into openHAB.
+
+## Supported Things
+
+The following devices are known to work with this binding:
+* iPhone 6s
+* iPhone 5c
+* iPhone 7
+* iPad Air 2/2017/Pro
+
+But other devices should work as well. Please provide feedback if you have tested another device type. We will add it to the list.
```

Comment:
It's often better to not start paragraphs with words like "But", "However", ...

Reply:
Changed - I´ll commit later today.